volume/drivers/proxy.go

```diff
@@ -0,0 +1,65 @@
+package volumedrivers
+
+// currently created by hand. generation tool would generate this like:
+// $ rpc-gen volume/drivers/api.go VolumeDriver > volume/drivers/proxy.go
+
+type volumeDriverRequest struct {
+	Name string
+}
+
+type volumeDriverResponse struct {
+	Mountpoint string `json:",ommitempty"`
+	Err        error  `json:",ommitempty"`
+}
+
+type volumeDriverProxy struct {
+	c client
+}
+
+func (pp *volumeDriverProxy) Create(name string) error {
+	args := volumeDriverRequest{name}
+	var ret volumeDriverResponse
+	err := pp.c.Call("VolumeDriver.Create", args, &ret)
+	if err != nil {
+		return err
```

Comment:
I think all these errors in proxy should be wrapped with some extra context, especially since they'll be coming from external sources.